.github/workflows/build-deploy-docs.yml

```diff
@@ -8,13 +10,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-python@v2
+      - uses: actions/setup-python@v4
         with:
           python-version: 3.x
       - name: Setup Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
-          node-version: 12
+          node-version: 18
```

Comment:
Admittedly, I have no way to confirm this workflow will still work after changing the node version to 18. On my local machine running Node 18, `npm run build` succeeds.